### ISSUE TITLE
请问能给出Co-Scale代码中主要函数库的版本嘛

### ISSUE THREAD

**Issue by liqiokkk** [lartpang/PyTorchCoding#5]
作者您好，请问能给出Co-Scale代码中主要函数库的版本嘛？

**Comment by lartpang** [comment]
我的实现中只涉及到了pytorch和einops。

实际上并没有用到依赖特定版本的那些功能。

```
pytorch                   1.8.1           py3.8_cuda10.2_cudnn7.6.5_0    pytorch
einops                    0.3.0
```

**Comment by liqiokkk** [comment]
好的，谢谢！
我使用pytorch是1.4版本，但没有跑通代码，所以猜测是版本原因。

**Comment by lartpang** [comment]
@liqi520 如果有问题，你可以贴上来，我这里是可以正常运行的。

**Comment by jpWang** [comment]
content_output = content_output.unflatten(dim=-1, sizes=(H, W))
报错应该是这行的问题，unflatten这个函数在老版本中不是这么用的。更新pytorch版本即可。

**Comment by lartpang** [comment]
@jpWang OK，多谢指出。我之前没有注意到这一点。我稍微更新下。

**Comment by liqiokkk** [comment]
我在pytorch1.2与1.4中都会报此错误，请问低版本中该函数如何实现？
module 'torch.nn.init' has no attribute 'trunc_normal_'
可以使用from timm.models.layers import trunc_normal_嘛？

**Comment by lartpang** [comment]
@liqi520 你可以比较下二者的代码，我觉得是一样的。
